plugins/chat/spec/system/transcript_spec.rb

```diff
@@ -173,14 +173,14 @@ def generate_transcript(messages, acting_user)
     context "when quoting a message in another message" do
       fab!(:message_1) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
 
-      xit "quotes the message" do
+      it "quotes the message" do
         chat_page.visit_channel(chat_channel_1)
 
         expect(chat_channel_page).to have_no_loading_skeleton
 
         clip_text = copy_messages_to_clipboard(message_1)
         click_selection_button("cancel")
-        chat_channel_page.send_message(clip_text)
+        chat_channel_page.send_message(clip_text.chomp)
```

Comment:
Wondering if this should happen in `#send_message` rather than in call sites, since it will always try to click the button. WDYT?